tests/lib/rules/no-trailing-spaces.js

```diff
@@ -19,8 +19,42 @@ var eslintTester = new ESLintTester(eslint);
 eslintTester.addRuleTest("lib/rules/no-trailing-spaces", {
 
     valid: [
-        "var a = 5;",
-        "var a = 5,\n    b = 3;"
+        {
+            code: "var a = 5;",
+            options: [{}]
+        },
```

Comment:
Lines 30-35 and 91-104 in tests/lib/rules/no-trailing-spaces.js

I'm afraid I can't spend any more time on this issue - please take over as you see fit.